### PR TITLE
Registration Page 

### LINKED ISSUE #21: FE: Register Wallet 2A

### PR DESCRIPTION
**Pull request by aidantrabs**
- very basic iteration, needs some tweaks
- closes #21 #22  #23 #24 #25 #26

### CODE REVIEW

File: src/components/TextInput.tsx [hunk 1]
```diff
@@ -1,43 +1,66 @@
...
+                        />
+                    ) : (
+                        <Input
+                            ref={ref as React.Ref<HTMLInputElement>}
+                            className={`${sharedClassNames} py-3`}
+                            invalid={!!error}
+                            required={required}
+                            {...inputProps}
+                            {...props}
+                        />
```
Comment (juancwu) [inline src/components/TextInput.tsx]: separate text area

**Comment by aidantrabs** [comment]
have a very basic iteration of the register wallet pages @juancwu @AmirAgassi 

- will need to update once backend is integrated
- also need to update the textinput + textarea for correct error formatting

File: src/components/TextInput.tsx [hunk 2]
```diff
@@ -6,43 +6,57 @@ interface TextInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
...
         return (
-            <div className="max-w-[400px] w-full">
+            <div className="w-full">
                 <Field>
                     {label && (
-                        <Label className="block text-2xl font-bold">
-                            {label}
-                        </Label>
+                        <div className="flex justify-between items-center mb-1">
+                            <Label className="block text-md font-bold">
```
Comment (AmirAgassi) [inline src/components/TextInput.tsx]:
```suggestion
                            <Label className="block text-md font-normal">
```

File: src/components/TextArea.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            min-h-[100px] resize-y
+            ${className}
+        `;
+
+        return (
+            <div className="w-full">
+                <Field>
+                    {label && (
+                        <div className="flex justify-between items-center mb-1">
+                            <Label className="block text-md font-bold">
```
Comment (AmirAgassi) [inline src/components/TextArea.tsx]:
```suggestion
                            <Label className="block text-md font-normal">
```

**Review by AmirAgassi — APPROVED** [review]
LGTM, but I lowkey think TextInput/TextArea's labels look better without bold

**Pull request merged**